packages/paste-website/src/components/tokens-list/token-card/index.tsx

```diff
@@ -47,7 +47,7 @@ export interface TokenCardProps {
   comment: string;
 }
 
-export const TokenCard: React.FC<TokenCardProps> = ({category, name, value, comment}) => {
+export const TokenCard: React.FC<TokenCardProps> = React.memo(({category, name, value, comment}) => {
```

Comment:
TODO: I think we can remove this now, since all of the props here have been flattened to be strings (or extensions of): 🎉